examples/ALB-ECS-WithElasticSearchLogs-NetworkStack

```diff
@@ -0,0 +1,540 @@
+import troposphere.elasticloadbalancingv2 as elb
```

Comment:
This file name should end with `.py` (`ALB-ECS-WithElasticSearchLogs-NetworkStack` -> `ALB-ECS-WithElasticSearchLogs-NetworkStack.py`)